File: app_mvp/src/main/java/com/example/jqorz/test1/mvp/presenters/MvpPresenter.java
package com.example.jqorz.test1.mvp.presenters;


import com.example.jqorz.test1.mvp.MvpView;

/**
 * Presenter的基础类，控制的MvpView的子类
 */
public interface MvpPresenter<V extends MvpView> {

    void attachView(V view);

    void detachView();

}

File: app_mvp/src/main/java/com/example/jqorz/mvptest/lib_kotlin/presenters/MvpPresenter.kt
package com.jqorz.tallybook.base.mvp.presenters

import com.jqorz.tallybook.base.mvp.MvpView

/**
 * Presenter的基础类，控制的MvpView的子类
 */
interface MvpPresenter<V : MvpView> {
    fun attachView(view: V)
    fun detachView()
}